reinforcement_learning/rl_cartpole_batch_coach/rl_cartpole_batch_coach.ipynb

```diff
@@ -0,0 +1,712 @@
+{
```

Comment:
<p>The learning agent *needs*.</p><p> </p><p>I would recommend adding links to the original cart pole example and the spot instance one. Also, as a large portion of the contents are identical across these three notebooks, it would be good to callout which part/sections readers should pay particular attention to.</p><p> </p><p>Also, shall we mention batch transformation in the introduction?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>